content/articles/building-a-lyrics-search-app-using-vanilla-javascript/index.md

```diff
@@ -0,0 +1,490 @@
+
+```css
+
+/**This is how placeholders are styled**/
+::placeholder {
+        color:gold;
+}
+
+```
+
+Next, style up the page description text (Learn your favorite song lyrics).
+
+```css
+
+/**Styling the intro text div which has the content of “learn your favorite song” lyrics**/
+
+.intro-text {
+      font-family: Verdana, Geneva, Tahoma, sans-serif;/**If the device doesn’t have any of the first three font-family sans-serif will automatically be used **/
+      padding:2rem 0;
+     Color:gold;
+}
+
+/**Styling the h1 header(learn your favourite):centering the h1 element**/
+h1 {
+      text-align:center;
+      padding:1rem 0;
+}
+
+/**Styling the h1 header(learn your favourite):centering the h1 element**/
+h2 {
+      text-align:center;
+      padding:1rem 0;
+}
+ /**You might decide to use the comma-separated form of styling tags with the same properties. I decided to be more explicit here for the sake of clarity**/
+```
+
+Centering the div container where the form and input field are embedded, using its id (lyrics-search).
+
+```css
+#lyrics-search {
+    text-align:center;
+    align-items:center;
+    margin:20px;
+    padding:20px;
+}
+
+```
+
+You will complete the css code for now by styling up the form and the input field.
+
+```css
+form {
+    margin:0 auto;
+    padding-left:10rem;
+}
+
+/**style the input field using its id(lyricSearch)**/
+
+#lyricSearch {
+        background-color: transparent;
+        color:#eeeeee;
+        outline:none;
+        height:30px;
+        width:64%;
+        margin:0 auto;
+        padding-right:2rem;
+        font-size:16px;
+        font-family:cursive;
+        outline-style:none;
+        border-top:none;
+        border-left:none;
+        border-right:none;
+        border-bottom:1px solid #eeeeee;
+}
+```
+
+You have successfully designed the web page by adding colors, font-sizes, font-family, padding, margin, and so on just to make this web app look nice.
+
+Here is what your web page should look like:
+![Screenshot for design](/building-lyrics-search-app-using-vanilla-javascript/design-screenshot.png)
+
+### Adding functionality with JavaScript
+
+In your `lyrics.js` file, declare variables and use the DOM selectors to connect with elements in the `lyrics.html` file, using the code snippet below:
+
+```JavaScript
+
+//you comment in js using double slash, with vscode just highlight what you want to comment and then press "ctrl + /"
+//defining variables
+const form = document.getElementById("searchMe");//target the form tag in the html file
+const search = document.getElementById("lyricSearch");//target th input field
+const output = document.getElementById("search-result");//target the output div
+
+```
+
+The desired API to get data (lyrics) for this web app to be fully functional, is the 
+[OVH API](https://api.lyrics.ovh).
```

Comment:
You can mention a brief overview about this API in the Introduction as well. Explain how it works in general.

Reply:
Add images or examples if necessary